### DISCUSSION TITLE
Regarding SunbirdRC backend installation

### DISCUSSION THREAD

**Discussion by ghost** [Sunbird-RC/community#589] [Issues]
Hello. I installed the SunbirdRC backend and watched a video (posted link below). I tried running compose.yaml and the rest of the included services are running, but keycloak and registry are not yet running. I've attached some screenshots of logs and service statuses. Please advise on how to resolve this.

-Thanks
![Screenshot from 2023-04-06 15-26-37](https://user-images.githubusercontent.com/109218913/230850922-54bcc05c-a3ff-4fac-830a-69b903315320.png)
![Screenshot from 2023-04-10 12-47-04](https://user-images.githubusercontent.com/109218913/230850924-f9ef8b9e-10e7-4486-8095-3c2c00a28511.png)
![Screenshot from 2023-04-10 12-47-32](https://user-images.githubusercontent.com/109218913/230850930-ce8109ea-245e-4e86-b4e9-fee1df40ee3c.png)
![Screenshot from 2023-04-10 12-47-51](https://user-images.githubusercontent.com/109218913/230850933-4db87f04-39f9-4a30-9260-c0bf29c72344.png)
![Screenshot from 2023-04-10 12-48-12](https://user-images.githubusercontent.com/109218913/230850936-7e231cb1-5b04-4279-928a-28bff647e508.png)
![Screenshot from 2023-04-10 12-59-14](https://user-images.githubusercontent.com/109218913/230850945-9f7ff681-3005-4c17-8624-87fb2b3c994e.png)
![Screenshot from 2023-04-10 12-59-32](https://user-images.githubusercontent.com/109218913/230850951-562f9171-3f08-4d62-9dcf-02e2414f539e.png)

Video link:
https://www.youtube.com/watch?v=C-cmtsEuPU0

**Comment by ghost** [comment]
@parthlawate @Renuka-S . Please advise.

**Comment by varadeth** [comment]
can you check if elastic search is running
and also if you can send out the logs for es and keycloak here

**Comment by ghost** [comment]
Attaching the logs.
[image: Screenshot from 2023-04-11 11-49-48.png][image: Screenshot from
2023-04-11 11-50-23.png][image: Screenshot from 2023-04-11 11-51-38.png]

**Comment by varadeth** [comment]
screenshots are not visible. could you please send that again

**Reply by ghost** [reply]
![Screenshot from 2023-04-11 11-49-48](https://user-images.githubusercontent.com/109218913/231075586-fcce7240-462e-4e03-a899-137027e6bfa0.png)
![Screenshot from 2023-04-11 11-50-23](https://user-images.githubusercontent.com/109218913/231075595-f2c364ec-490e-4383-8f58-515e20b41943.png)
![Screenshot from 2023-04-11 11-51-38](https://user-images.githubusercontent.com/109218913/231075597-d69707bd-24d6-48fe-956a-b65feb0e9f52.png)

**Comment by varadeth** [comment]
i had a call with him. we did complete setup. we did not issue any problems which was mentioned in the discussion
